docs/root/intro/arch_overview/init.rst

```diff
@@ -11,7 +11,7 @@ accepting new connections.
   :ref:`CDS <arch_overview_dynamic_config_cds>` if applicable, waits for one response (or failure),
   and does the same primary/secondary initialization of CDS provided clusters.
 * If clusters use :ref:`active health checking <arch_overview_health_checking>`, Envoy also does a
-  single active HC round.
+  single active healthcheck round.
```

Comment:
Please be consistent everywhere around "health check" (no dash, space between words). Applies elsewhere.

Reply:
ok, i will do that